### PR TITLE
Update explainer for unsized media 

### PR DESCRIPTION
**Pull request by loonybear**
With respect to intrinsicSize

### CODE REVIEW

**Comment by yoavweiss** [comment]
@loonybear Can you please [join](https://www.w3.org/community/wicg/participants) the WICG in order to appease the IPR bots? Thanks! :)

**Comment by loonybear** [comment]
Marked as non substantive for IPR from ash-nazg.

**Comment by loonybear** [comment]
@clelland PTAL

File: policies/unsized-media.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 loonybear@, last updated: 4/25/2018
 
-<span style="color:#38761d;">Status: Draft Proposal</span>
+<span style="color:#38761d;">Status: feature landed behind a flag</span>
```
Comment (clelland) [inline policies/unsized-media.md]: Can you change this to say that it is available "in Chrome" behind a flag? -- The feature itself is still just a proposal from a standards perspective, even if it has an experimental implementation in one browser right now.

**Comment by loonybear** [comment]
Done. PTAL

**Comment by clelland** [comment]
Merged! Sorry for the wait.

**Pull request merged**